Include dependency review GH

naveensrinivasan opened the pull request:
The dependency review action in your repository to enforce dependency reviews on your pull requests. The action scans vulnerable versions of dependencies introduced by package version changes in pull requests and warns you about the associated security vulnerabilities. 

https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-dependency-review#dependency-review-enforcement

naveensrinivasan committed:
Dependency review

rvema reviewed (changes requested):
Is it possible to skip this check on Markdown files?

naveensrinivasan committed:
Improve Dependency Review Workflow

- Update the Github Action in `.github/workflows/dependency-review.yml` to include changes in paths other than `.md`

Signed-off-by: naveensrinivasan <172697+naveensrinivasan@users.noreply.github.com>

naveensrinivasan commented:
> Is it possible to skip this check on Markdown files?

Thanks. 👍

rvema reviewed (approved):
lgtm
